product-service/createProduct.js

```diff
@@ -0,0 +1,35 @@
+import { headers } from './constants';
+import { client } from './db';
+
+export const create = async (event) => {
+  console.log(event);
+
+  try {
+    await client.query('BEGIN');
+
+    const { title, description, price } = JSON.parse(event.body);
+    const addProductQuery = 'insert into products (title, description, price) values ($1, $2, $3) returning *';
+    const { rows } = await client.query(addProductQuery, [title, description, price]);
+    const addToStockQuery = 'insert into stocks(product_id, count) values ($1, $2)';
+    const productId = rows[0].id;
+
+    await client.query(addToStockQuery, [productId, 1]);
+    await client.query('COMMIT');
```

Comment:
Would be nice to move all db-specific logic outside the lamdas.

Reply:
Good point